src/routes/workshop-a11y-fundamentals.svelte

```diff
@@ -203,174 +550,348 @@
+            Codebase with all the mentioned resources;
+          </li>
+          <li>
+            Exercises, including the solutions explained;
+          </li>
+         </ul>       <!-- TODO list -->
+        </Accordion>
+      </li>
+
+      <li>
+        <Accordion summary="I know A11Y. Will I learn something new?">
+          Oh, yeah. Even me, every time I give this workshop I always learn something new. Unless you are an A11Y expert, I’m confident that you'll fill some gap.
+        </Accordion>
+      </li>
+
+      <li>
+        <Accordion summary="Is the workshop accessible?">
+          The video meeting can be captioned and the exercises briefings are fully accessible. If you have any accessibility concern, please reach out to me, and we can arrange it together.
+        </Accordion>
+      </li>
+
+      <li>
+        <Accordion summary="What screen reader will I use?">
+                        <!-- TODO list -->
+
+          You can use the one that suits you the best. I own a Mac, which means I’ll be using Voice Over.
+          During an online workshop it's not practical to help everyone using a SR (screen reader). For that reason, please take some minutes to practice in advance.
+          <ul class="t-list">
+            <li>
+              Mac: You'll be using VoiceOver. Watch this VO introduction.
+            </li>
+            <li>
+              Windows: Install NVDA and watch this NVDA introduction.
+            </li>
+            <li>
+              Linux: Install Orca and watch this Orca introduction.
+            </li>
+            <li>
+              SR keyboard shortcuts: VO and NVDA and Orca.
+            </li>
+          </ul>
+          A few days before the workshop, I will send you these guides for you to practice.
+        </Accordion>
+      </li>
+
+      <li>
+        <Accordion summary="Is there any discounts available?">
+          The workshop will be in Early Bird price for a week. After that, there won’t be discounts available. I understand that the cost of this workshop can be too high for some people. If you are a student you can reach out to me.
+        </Accordion>
+      </li>
+
+      <li>
+        <Accordion summary="Can I buy a ticket to my team?">
+          You can but keep it mind that 1 ticket is for 1 
+            <!-- TODO CLARIFY THIS -->
+          person/seat. Besides these public events, I also provide customised trainings. You can contact me to discuss private sessions options.
+        </Accordion>
+      </li>
+
+      <li>
+        <Accordion summary="Can I ask you questions after the workshop?">
+          Of course! During the workshop, I'll invite to join a Discord community with other attendees to clarify any question.
+        </Accordion>
+      </li>
+
+      <li>
+        <Accordion summary="Is there a Code of Conduct?">
+          Yes, mine is pretty simple: Be kind and treat each other with respect and understanding. There’s zero tolerance for unkind behavior.
```

Comment:
Yes, and it's pretty simple: be kind and treat each other with respect and understanding. There’s zero tolerance for unkind behavior.